spec/models/tag_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Tag do
+
+  it "decreases tag_count when a debate is hidden" do
+    debate = create(:debate)
+    tag = create(:tag)
```

Comment:
Hi! This code has been translated to consul from the Madrid branch, "as it was". Yes, using `let` here would make the code less repetitive. Do you want to send a pr :)?